.azuredevops/pipelines/DirectXTex-GitHub-Dev17.yml

```diff
@@ -39,236 +39,116 @@ pool:
 
 jobs:
 - job: DESKTOP_BUILD
-  displayName: 'Win32 Desktop'
+  displayName: 'Windows Desktop'
   timeoutInMinutes: 120
   cancelTimeoutInMinutes: 1
+  strategy:
+    maxParallel: 3
```

Comment:
The official ADO MS-Hosted policy is that you get up to 10 parallel agents at a time for public projects. I throttled my pipelines a bit to keep it closer to 4 or 5 at a time.